packages/core/src/domain/document-signing/bulk-sign.ts

```diff
@@ -0,0 +1,136 @@
+import axios from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { DocumentReference } from "@medplum/fhirtypes";
+import { capture } from "../../util/capture";
+import { S3Utils } from "../../external/aws/s3";
+import { DocumentReferenceWithURL } from "../document-bulk-signer-response";
+import { searchDocuments } from "../../external/fhir/document/search-documents";
+import { MetriportError } from "../../util/error/metriport-error";
+import { BulkGetDocUrlStatus } from "../bulk-get-document-url";
+
+const ossApi = axios.create();
+dayjs.extend(duration);
```

Comment:
nit: I suggest we keep calls to `dayjs.extend()` before any const so we avoid using the extension before it gets initialized.